bpf/Makefile.bpf

```diff
@@ -76,8 +76,9 @@ endif
 checkpatch:
 	@$(ECHO_CHECK) "(checkpatch)"
 	$(QUIET) $(CONTAINER_ENGINE) container run --rm \
-		--workdir /workspace \
-		--volume $(CURDIR)/..:/workspace \
+		--workdir $(realpath $(CURDIR)/..) \
+		--volume ~/.gitconfig:/.gitconfig \
```

Comment:
It'd be nice if we could just rely on configuring any relevant settings inside the container, since we control it, rather than pushing more host user content into the container.

Reply:
Issued https://github.com/cilium/image-tools/pull/221

Reply:
This adds `/.gitconfig` with a wildcard safe directory. Will update this PR once that one is merged and built.